app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberController.scala

```diff
@@ -56,23 +60,30 @@ class EnterDuplicateMovementReferenceNumberController @Inject() (
   def enterDuplicateMrn(): Action[AnyContent] =
     authenticatedActionWithSessionData.async { implicit request =>
       withDuplicateMovementReferenceNumberAnswer { (_, _, answers) =>
+        val messagesKey = resolveMessagesKey(featureSwitch)
         answers.fold(
           ifIncomplete =>
             ifIncomplete.duplicateMovementReferenceNumber match {
```

Comment:
the Incomplete/Complete model needs to be replaced with the simplified model

Reply:
Done